packages/@okta/vuepress-site/.vuepress/scripts/build-replacement-vars.js

```diff
@@ -0,0 +1,26 @@
+const thingsToReplace = { 
+  /* KEYS HERE GET WRAPPED IN '-=OKTA_REPLACE_WITH_XXX=-'
+   *
+   * Changes WILL require restarting `yarn dev` :(
+   */
+
+/* VALUES BELOW THIS LINE INTENDED FOR EASY EDITING */
+  WIDGET_VERSION: 3, // Leave as major version
+  TEST_JUNK: 'this is a test replacement', // Leave for testing
```

Comment:
Can these be pulled in via `/.vuepress/config.js`?

Reply:
Can do.